components/script/dom/element.rs

```diff
@@ -137,6 +137,8 @@ pub struct Element {
     /// when it has exclusive access to the element.
     #[ignore_heap_size_of = "bitflags defined in rust-selectors"]
     selector_flags: Cell<ElementSelectorFlags>,
+    #[ignore_heap_size_of = "RwLock"]
+    cached_lang: RwLock<Option<Atom>>,
```

Comment:
Ah right, that is supposed to be invalidated if the attribute's value is mutated, and should be handled in `attribute_mutated`. My current PR still doesn't have any of those changes, but will push a fix soon.